web/src/app/modules/shared/components/presentation/preferences/preferences.component.html

```diff
@@ -60,6 +60,11 @@ <h4>{{ section.name }}</h4>
                         </div>
                       </div>
                     </ng-container>
+                    <ng-container *ngSwitchCase="'dropdown'">
+                      <label>{{ element.label }}</label>
+                      <app-view-dropdown [view]="element" (selectedValue)="onDropDownValueChange($event, element.name)">
```

Comment:
Why is dropdown used here over select (https://clarity.design/angular-components/select/)?

Reply:
This is a good question, we are not using clarity here, I saw this component and just used

Reply:
I am OK with using our dropdown here, there a lots of benefits of dogfooding.

Reply:
Implicitly this means using dropdown over select for all of preferences in order to maintain a consistent design. Are we okay with that?

Reply:
I think we should be consistent within the Clarity guidelines.

https://clarity.design/angular-components/dropdown/#dropdown-vs-select